Formula/pegtl.rb

```diff
@@ -14,20 +14,23 @@ class Pegtl < Formula
   end
 
   depends_on "cmake" => :build
+  # Fix `error: 'path' is unavailable: introduced in macOS 10.15`
+  depends_on macos: :catalina
```

Comment:
Shouldn't that be catalina or higher?

Reply:
yeah, you are right, I was confused about the right syntax.

Let me try some other solution for solving Mojave build. :)